### ISSUE TITLE
Player count set to 1 but console reports "SV: maxplayers set to 32"

### ISSUE THREAD

**Issue by JamesC01** [Facepunch/sbox-issues#782]
**Describe the bug**
A random person joined one of my games. I thought maybe I forgot to put it on 1 player. I made another server, and checked the console, and it reports SV: maxplayers set to 32.

**To Reproduce**

Maybe a 1 player game, and check the console. It's near the start, if you clear it before starting a server.

**Expected behavior**
maxplayers should be set to the player count.

**Comment by Gmod4phun** [comment]
Basically a duplicate of https://github.com/Facepunch/sbox-issues/issues/8

**Comment by robotboy655** [comment]
Duplicate of #8
